src/odemis/driver/technolution.py

```diff
@@ -1293,16 +1311,19 @@ def system_checks(self, ASM):
         :param ASM (AcquisitionServer object) : AcquisitionServer object used for changed the state of the system and
         checking system parameters.
         """
+        # Try statements are used to prevent triggering an error in an error.
         try:
-            response = ASM.asmApiGetCall("sam_connection_operational", 200)
+            item_name = "sam_connection_operational"
```

Comment:
Can we also resolve this issue since with the adding of the dedicated getSysSwVersion etc. this is moved? These checks are only there for error checking.

Reply:
@Sabrina85 With the specs of sim3 this conversation is resolved isn't it?

Reply:
yes